tools/dia_dll.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python
+# Copyright (c) 2012 The Chromium Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+"""
+This script is based on chromium/chromium/master/tools/clang/scripts/update.py.
+
+It is used on Windows platforms to copy the correct msdia*.dll to the
+clang folder, as a "gclient hook".
+"""
+
+import os
+import shutil
+import stat
+import sys
+
+
+# Path constants. (All of these should be absolute paths.)
+THIS_DIR = os.path.abspath(os.path.dirname(__file__))
+LLVM_BUILD_DIR = os.path.abspath(os.path.join(THIS_DIR, '..', '..', 'third_party',
+                                              'llvm-build', 'Release+Asserts'))
+
+
+def GetDiaDll():
+  """Get the location of msdia*.dll for the platform."""
+
+  # Bump after VC updates.
+  DIA_DLL = {
+    '2013': 'msdia120.dll',
+    '2015': 'msdia140.dll',
```

Comment:
Do we support 2013 or 2015?

Reply:
I don't think so, but the upstream script had it and I figured it may be helpful when trying to figure out how to add new ones.

Reply:
Well, the upstream script has it because they have supported these versions in the past. I was thinking leave the map construction here even though it's not currently needed, to make it easy to change in the future, but remove the old entries that we will never use.